README.md

```diff
@@ -67,23 +67,25 @@ At the bottom, in the `Command Preparations` section, you will press the `+Add`
 In the first text box the `config.do_cmd` column, you will write:
 
 ```
-cmd /C powershell.exe -File %PATH_TO_THIS_REPOSITORY%\setup_sunvdm.ps1 %SUNSHINE_CLIENT_WIDTH% %SUNSHINE_CLIENT_HEIGHT% %SUNSHINE_CLIENT_FPS% %SUNSHINE_CLIENT_HDR% > %PATH_TO_THIS_REPOSITORY%\sunvdm.log 2>&1
+cmd /C powershell.exe -executionpolicy bypass -windowstyle hidden -file "%PATH_TO_THIS_REPOSITORY%\\setup_sunvdm.ps1" %SUNSHINE_CLIENT_WIDTH% %SUNSHINE_CLIENT_HEIGHT% %SUNSHINE_CLIENT_FPS% %SUNSHINE_CLIENT_HDR% "%VDD_NAME%" > "%PATH_TO_THIS_REPOSITORY%\\sunvdm.log" 2>&1
```

Comment:
hey! I think that you add doubles //, and it could break the script
(using your sunshine AIO, and found some issues whit this, great work LeGeryChEeSe! tbh)

Reply:
Thank you! I have updated the README.md with the last commit.